package.json

```diff
@@ -48,8 +48,8 @@
         "dev": "lerna run --parallel --scope '@padloc/{server,pwa}' --parallel dev",
         "tauri:dev": "lerna run --parallel --scope '@padloc/{server,tauri}' --parallel dev",
         "tauri:update": "lerna run update",
-        "tauri:build": "lerna run build:production",
-        "tauri:build:debug": "lerna run build --scope @padloc/tauri",
+        "tauri:build": "lerna run build --scope @padloc/tauri",
+        "tauri:build:debug": "lerna run build:debug",
```

Comment:
When we replaced these in `package/tauri/package.json`, we forgot to replace them here too.